.github/workflows/deploy-aggregate.yaml

```diff
@@ -33,6 +33,7 @@ on:
   push:
     branches:
       - main
+      - deploy-bug-fix
```

Comment:
```suggestion
```

Reply:
Yeah just need that cleanup. Honestly I have no idea why. I just kept running into not enough memory issues on deploy for some reason.